src/js/button.js

```diff
@@ -26,24 +26,14 @@ class Button extends ClickableComponent {
    * @method createEl
    */
   createEl(tag = 'button', props = {}, attributes = {}) {
+    if (tag !== 'button') {
+      throw new Error('Buttons must be created with a "button" tag.');
```

Comment:
I think the reason for this was originally backwards compatibility. I wonder if we can now just ignore the passed in tag and and always create a `button`. We'd want to still log the warning, though. So, maybe just throwing is fine :)

@OwenEdwards thoughts?